tests/test_hls_report.py

```diff
@@ -108,12 +93,27 @@ def refine(res_tbl):
     res[0] = res[0].replace(', ', '', 1)
     return res
 
-def get_expected(wd):
+def get_expected(ver, wd):
```

Comment:
What's the difference between `Normal` and `Same`? Just test cases for different apps?

Reply:
Yes. I can try merging those into the same function to simplify things now that I think of it.